lib/workers/global/config/parse/cli.ts

```diff
@@ -34,6 +34,9 @@ export function getConfig(input: string[]): AllConfig {
         .replace('--aliases', '--registry-aliases')
         .replace('--include-forks=true', '--fork-processing=enabled')
         .replace('--include-forks', '--fork-processing=enabled')
+        .replace('--recreate-closed=false', '--recreate-when=auto')
+        .replace('--recreate-closed=true', '--recreate-when=always')
+        .replace(regEx(/--recreate-closed$/), '--recreate-when=always')
```

Comment:
```suggestion
        .replace('--recreate-closed', '--recreate-when=always')
```
should work for simple string replace too